src/hooks/useProjects/useProjects.ts

```diff
@@ -0,0 +1,36 @@
+import { useCallback, useEffect, useState } from "react";
+import { database } from "services";
+import { IProject } from "types";
+
+export const useProjects = () => {
```

Comment:
This file should managed handlers for common operations from `services/project`:

- getProjects
- getProject
- addProject
- updateProject
- deleteProject
- increaseRow
- increaseRepeat
- decreaseRow
- decreaseRepeat